ts_scripts/install_dependencies.py

```diff
@@ -55,6 +55,10 @@ def install_python_packages(self, cuda_version, requirements_file_path):
         # developer.txt also installs packages from common.txt
         os.system(f"{sys.executable} -m pip install -U -r {requirements_file_path}")
         # If conda is available install conda-build package
+        if nightly:
```

Comment:
Can we if-else this with line 53? Otherwise, we install torch and then reinstall as nightlies.

Reply:
Not sure I can unfortunately without significantly changing how our requirements.txt is setup - a lot of them explicitly pick a torch version

Reply:
I see, remember we had this discussion before.... then maybe just clarify in a comment why and vow to improve this in the future and then see you in clowntown! :-D